pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -762,6 +763,83 @@ func parseSystemAdapterConfig(config *zconfig.EdgeDevConfig,
 	log.Infof("parseSystemAdapterConfig: Done")
 }
 
+var deviceIoListPrevConfigHash []byte
+
+func parseDeviceIoListConfig(config *zconfig.EdgeDevConfig,
+	getconfigCtx *getconfigContext, forceParse bool) {
```

Comment:
Please remove the forceParse argument since you are not using it (always false)

Reply:
done..